cmd/kubeadm/app/cmd/init.go

```diff
@@ -33,7 +33,7 @@ import (
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/discovery"
 	kubemaster "k8s.io/kubernetes/cmd/kubeadm/app/master"
-	"k8s.io/kubernetes/cmd/kubeadm/app/phases/apiconfig"
+	apiconfigphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/apiconfig"
```

Comment:
Idk what's best, here I just aligned it with the other phase packages that have the `phase` suffix.
I don't feel strongly for either case.

@pires What do you think is the most clean option?